test/unit/resources/ssh_key_test.rb

```diff
@@ -17,15 +17,6 @@ def teardown
     FileUtils.rm_rf("#{@private_key_path}.pub")
   end
 
-  def test_ssh_key_resoure
```

Comment:
this entire file should have been removed part of https://github.com/inspec/inspec/pull/7248 or any other SSH resource removal PR

Reply:
Yes but since it is not merged yet.. have made this as a temporary change.. once merged this would go on rebase.

Reply:
it is merged now @Nik08